coralproject/talk
# [CORL-647] Comment Action Fix

Adjusts the query for the comment action creation to include the tenant ID to encourage index use.

## Verification

1. Checkout `master`
2. Using a MongoDB console (via `docker run --rm -ti --link mongo:mongo mongo:4.2 mongo mongodb://mongo:27017/coral`) run `db.setProfilingLevel(2)`
3. React to a comment
4. Run `db.system.profile.find({"ns": "coral.commentActions", "command.findAndModify": "commentActions"}, {"planSummary": 1})` and notice that it is performing a `COLLSCAN` (that's bad!)
5. Switch to the `bugfix/CORL-647` branch
6. Run `db.setProfilingLevel(0); db.system.profile.drop(); db.setProfilingLevel(2)`
7. React to another comment
8. Run `db.system.profile.find({"ns": "coral.commentActions", "command.findAndModify": "commentActions"}, {"planSummary": 1})` and notice that it is performing a `IXSCAN` (that's good!)
